AutomatedTesting/Gem/PythonTests/EditorPythonBindings/tests/Editor_ObjectManagerCommands_Works.py

```diff
@@ -0,0 +1,120 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+def check_result(result, msg):
+    from editor_python_test_tools.utils import Report
+    if not result:
+        Report.result(msg, False)
+        raise Exception(msg + " : FAILED")
```

Comment:
I see this same function defined earlier. Maybe move the function entirely into editor_python_test_tools